packages/schematics/angular/service/index.ts

```diff
@@ -63,23 +63,25 @@ function addProviderToNgModule(options: any): Rule {
 }
 
 export default function (options: any): Rule {
-  options.module = normalizePath(options.module);
+  return (host: Tree, context: SchematicContext) => {
+    options.module = findModuleFromOptions(host, options);
```

Comment:
This should only be called if `options.module` has a value...services are not automatically provided.